classes/03class/exercises/c03-compose01/frdvo/docker-compose.yaml

```diff
@@ -0,0 +1,47 @@
+version: "3.8"
+
+services:
+
+  web:
+      build: .
+      ports:
+        - "8100:80"
+      networks:
+        - front-end
+        - back-end
+      volumes:
+        - ./src:/var/www/html
+      env_file:
+        - '.web.env'
+      deploy:
+        resources:
+          limits:
+            cpus: '0.50'
+            memory: 50M
+          reservations:
+            cpus: '0.25'
+            memory: 20M
+      depends_on: 
+      - db
+
+
+  db:
+      image: "mysql:5.7"
+      ports:
+        - "3306:3306"
```

Comment:
You don't need to expose port 3306 to the host.

The web service can reach the `db` container through the internal network as Docker has a service discovery by default. 

So you can just point the DB_HOST in your `web` service as:
```
        environment:
            DB_HOST: db
```